cdap-unit-test/src/main/java/co/cask/cdap/test/UnitTestManager.java

```diff
@@ -86,6 +91,24 @@
 public class UnitTestManager implements TestManager {
 
   private static final Gson GSON = new Gson();
+  private static final ClassAcceptor CLASS_ACCEPTOR = new ClassAcceptor() {
```

Comment:
should we define this in the interface (``TestManager``) if it is used by both ``UnitTestManager`` and ``IntegrationTestManager``?